tests/conftest.py

```diff
@@ -82,3 +84,8 @@ def profile_test_client():
     with TestClient(app) as client:
         yield client
     app.dependency_overrides = {}
+
+
+@pytest.fixture
+def Calendar():
```

Comment:
Maybe in a more specific file?

Reply:
Its a very general object I guess others will want to find and use it as well in the future.

Reply:
I meant in the same file where user exists (not in this one).